lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -253,6 +253,7 @@ sub start
         send_federation       => ( not $self->{dendron} ),
         update_user_directory => ( not $self->{dendron} ),
         enable_media_repo     => ( not $self->{dendron} ),
+        run_background_tasks_on  => ( $self->{dendron} ? "background_worker1" : "master" ),
```

Comment:
I'm not fully confident on what `self->{dendron}` is, but it seems to be only exist when workers are being used.

Reply:
Yeah, its basically the thing that enables `pydron` and stuff